felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/AktiveArbeidsgivere.kt

```diff
@@ -0,0 +1,15 @@
+package no.nav.helsearbeidsgiver.felles
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class AktiveArbeidsgivere(
+    val fulltNavn: String? = null,
```

Comment:
I hvilke tilfeller mangler arbeidsgivere navn, men de har underenheter? Er det sånn fordi vi henter navnet og underenhetene fra to ulike steder (og f.eks. stedet vi henter navn fra kan være nede)?

Reply:
Denne må jeg også passere til @b162214.